lib/cdo/only_one.rb

```diff
@@ -9,7 +9,8 @@ def only_one_running?(script)
     rescue Errno::ESRCH
       File.delete(pidfile)
       exists = false
-    rescue ::Exception   # for example on EPERM (process exists but does not belong to us)
+    rescue ::Exception => e # for example on EPERM (process exists but does not belong to us)
+      $stderr.puts "PID file #{pidfile} exists and process could not be killed: #{e}"
```

Comment:
Note, we're not killing the process here. That would somewhat defeat the purpose, as this is meant to leave the old one running and abort starting the new (this) one. Instead we're sending the null (0) signal, which just checks to see if that pid still exists. 

I still like logging something here so we have a record of this unexpected exception, I'd just remove the "could not be killed part". As I'm sure you know (stated here for the record), logging to stderr here from a cronjob will [notify](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/cronjob#L70) honeybadger 👍 

I wonder if after reboot we got unlucky and the same pid was reused for a new unrelated process. That could  explain why we'd end up in this rescue block. 

In addition to this logging, it really would be nice to make sure these files are cleaned up on reboot.